---
source: tests/test_runner.rs
expression: stripped_actual_output.trim()
---
============
INPUT
============
let foo
  : Int -> Int
  = \0 -> 0
    \1 -> 1
    \1 -> 1
    \_ -> 2

============
EXIT CODE
============
1

============
STDOUT
============


============
STDERR
============
error:
  ┌─ tests/compiler/typecheck/unreachable_case_integer_1/unreachable_case_integer_1.kk:5:6
  │
5 │     \1 -> 1
  │      ^ Unreachable case
  │
  = This case is unreachable as it is already handled by previous cases.
